examples/fine-tuning/create_dataset.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+if [ "$#" -ne 2 ] || ! [ -d "$2" ]; then
+  echo "Usage: $0 <NEW_DATASET_DIR> <MNIST_DIR>" >&2
+  exit 1
+fi
+
+NEW_DATASET_DIR=$1
+MNIST_DIR=$2
+
+# create new dataset directories
+mkdir -p $NEW_DATASET_DIR/odd
+mkdir -p $NEW_DATASET_DIR/even
+
+# create symbolic links
+for digit in 0 2 4 6 8
+   do cp $MNIST_DIR/$digit/*.png $NEW_DATASET_DIR/even/
```

Comment:
Oh I see, you're right. [The max_per_category check](https://github.com/NVIDIA/DIGITS/blob/v3.0.0-rc.3/tools/parse_folder.py#L393-L397) comes before [the shuffling](https://github.com/NVIDIA/DIGITS/blob/v3.0.0-rc.3/tools/parse_folder.py#L417).

Maybe I didn't notice it because I used the MNIST test images, not the training images?